db/migrate/20210428103430_create_active_storage_tables.active_storage.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+class CreateActiveStorageTables < ActiveRecord::Migration[5.2]
```

Comment:
Do we want the version to be 5.2 here? It's 6.1 in other migration

Reply:
Where did we take this from, out of curiosity?

Reply:
just ran bin/rails active_storage:install from: https://edgeguides.rubyonrails.org/active_storage_overview.html